### PR TITLE
docs(mediaObserver): create new mediaObserver documentation to replace

### PR DESCRIPTION
**Pull request by DuncanFaulkner**
the ObservableMedia documentation should also be added to the WIKI pages and the ObservableMedia page deprecated or removed.

### CODE REVIEW

File: docs/documentation/mediaObserver.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+```typescript
+asObservable(): Observable<MediaChange>
+```
+
+Use the `.asObservable()` accessor method to access the **Observable** and use **RxJS** operators; such as `media.asObservable().filter(....)`.
+
+> Do not forget to **import** the specific RxJS operators you wish to use!
+
+```typescript
+import { Component, OnDestroy, OnInit } from "@angular/core";
```
Comment (CaerusKaru) [inline docs/documentation/mediaObserver.md:36]: Single quotes throughout please

File: docs/documentation/mediaObserver.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  selector: "app-root",
+  templateUrl: "./app.component.html",
+  styleUrls: ["./app.component.css"],
+})
+export class AppComponent implements OnInit, OnDestroy {
+  title = "Angular Flex-Layout";
+
+  constructor(private mediaObserver: MediaObserver) {}
+  private mediaSubscription!: Subscription;
+  private activeMediaQuery: string = "";
```
Comment (CaerusKaru) [inline docs/documentation/mediaObserver.md:51]: Redundant type parameter

**Review by CaerusKaru — CHANGES_REQUESTED** [review]
Looks good overall! Just a couple of nits then should be good to go

File: docs/documentation/mediaObserver.md (hunk 2 above)
Comment (DuncanFaulkner) [reply docs/documentation/mediaObserver.md:51]: I assume this is the title? sorry I meant to remove that before doing the PR

File: docs/documentation/mediaObserver.md (hunk 1 above)
Comment (DuncanFaulkner) [reply docs/documentation/mediaObserver.md:36]: Fixed double-quote issue by adding prettierrc.js

**Review by DuncanFaulkner — COMMENTED** [review]
fixed in updated PR

**Review by DuncanFaulkner — COMMENTED** [review]
set prettier to singleQuotes

File: docs/documentation/mediaObserver.md [hunk 3]
```diff
@@ -122,15 +120,15 @@ For example:
...
+import { MediaChange, MediaObserver } from '@angular/flex-layout';
 
-const PRINT_MOBILE = "print and (max-width: 600px)";
+const PRINT_MOBILE = 'print and (max-width: 600px)';
 
 @Component({
-  selector: "responsive-component",
+  selector: 'responsive-component',
   template: `
-    <div class="ad-content" *ngIf="media.isActive('xs')">
```
Comment (CaerusKaru) [inline docs/documentation/mediaObserver.md]: Div quotes should be double, right?

File: docs/documentation/mediaObserver.md (hunk 2 above)
Comment (CaerusKaru) [reply docs/documentation/mediaObserver.md]: No, I mean `string` is unnecessary when you are assigning it to the empty string.

**Comment by CaerusKaru** [comment]
Not really sure why CI is just hanging like this...

**Review by DuncanFaulkner — COMMENTED** [review]
sorry that probably shouldn't have been included in the PR, VS Code was swapping single quotes for double quotes and I was having an issue stopping it swapping. 
Happy to delete this file as I don't think this is required,

**Comment by DuncanFaulkner** [comment]
> Not really sure why CI is just hanging like this...
Yes not sure what's going on with this

File: docs/documentation/mediaObserver.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+import { distinctUntilChanged } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css'],
+})
+export class AppComponent implements OnInit, OnDestroy {
+  constructor(private mediaObserver: MediaObserver) {}
+  private mediaSubscription!: Subscription;
```
Comment (CaerusKaru) [inline docs/documentation/mediaObserver.md:48]: Shouldn't the exclamation point be at the tail end? E.g. `private mediaSubscription: Subscription !;`

**Comment by CaerusKaru** [comment]
Sorry for the nitpicks that keep coming, I promise that's the last one. Once it's resolved, can you close this PR and open a new one to trigger CI? If that doesn't work I'll open a PR on your behalf (so you still get credit for the commit), and then we can merge it.

File: docs/documentation/mediaObserver.md (hunk 4 above)
Comment (DuncanFaulkner) [reply docs/documentation/mediaObserver.md:48]: It's the non-null-assertion operator, and it's not required here (tbh, I can't remember adding it). I will amend the PR.

**Comment by DuncanFaulkner** [comment]
Will open new PR in hope that CI runs!

**Pull request closed without merging**